java-client/src/main/java/org/opensearch/client/opensearch/indices/IndexSettingsIndexing.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
@russcam could you please remove the `Elasticsearch` license headers in all new files? (unless those where copied from Elasticseach) , thank you

Reply:
They're a copy of the following, with modifications to model them better than what is auto-generated:

- [indexing](https://github.com/elastic/elasticsearch-java/blob/main/java-client/src/main/java/co/elastic/clients/elasticsearch/indices/IndexingSlowlogSettings.java#L52)
- [search](https://github.com/elastic/elasticsearch-java/blob/main/java-client/src/main/java/co/elastic/clients/elasticsearch/indices/SettingsSearch.java#L48)

Reply:
Thanks @russcam , all good than